### DISCUSSION TITLE
[Bug]: Buscas com caracteres variantes como á, é, í, não retornam o resultado esperado

### DISCUSSION THREAD

**Discussion by Honokai** [Power-Components/livewire-powergrid#805] [Ideas]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

No, this error is not related to views.

### Is there an error in the console?

No

### PowerGrid

3.1.0

### Laravel

8.83.16

### Livewire

2.10.5

### Theme

Tailwind 3.x

### Describe the bug.

Pesquisas/filtros pela coluna do powergrid não retornam os resultados esperados quando a palavra possui acentuação e o usuário escreve sem a acentuação. Exemplo: 

Em meu banco de dados temos ÁLCOOL.

1- Quando pesquiso por alcool (sem acento), o powergrid não retorna nenhum registro de álcool (com acento).

2- Quando pesquiso por tubulacao, o powergrid não retorna nenhum registro de "tubulação";

Enquanto que se realizarmos uma consulta sql diretamente, no servidor com configurações padrões, o resultado da consulta entrega todas as variações, seja Tubulacao, Tubulação, Tubulaçao, etc.

Seria possível uma correção para conversão dessas variantes para suas versões normais? 

Exemplo:

```php
class Collection implements CollectionFilterInterface
{

...

 private array $dicionario = [
  "caracteres" => [
    "á",
    "é",
    "í",
    "Á",
    "É",
    "Í"
    ...
  ],
  "substitutos" => [
    "a",
    "e",
    "i",
    "A",
    "E",
    "I",
    ...
  ]
]

...

case 'contains':
    $this->query = $this->query->filter(function ($row) use ($field, $value) {
        $row = (object) $row;

        // return false !== stristr($row->{$field}, strtolower((string) $value));
        return false !== stristr(str_replace($this->dicionario['vogals'], $this->dicionario['substituir'], $row->{$field}), strtolower(str_replace($this->dicionario['vogals'], $this->dicionario['substituir'], (string) $value)));
    });

    break;
```

**Comment by luanfreitasdev** [comment]
What database version are you using?.

What happens if you make a filter like this?

```sql
select * from `dishes` where ((`dishes`.`id` LIKE '%Nabia%' or `dishes`.`name` LIKE '%Nabia%' or `dishes`.`chef_name` LIKE '%Nabia%' ) or exists (select * from `categories` where `dishes`.`category_id` = `categories`.`id` and `name` LIKE '%Nabia%')) and `dishes`.`deleted_at` is null order by `dishes`.`id` asc limit 10 offset 0
```

In mine it works with MySQL 8

**Reply by Honokai** [reply]
Sorry, the problem was how I wrote my datasource.
